### PR TITLE
Set up the assignment

### PR DESCRIPTION
**Pull request by durdonadev**

### CODE REVIEW

**Comment by aliramazon** [comment]
Overal, 3 commits is good.

File: assignments/assignment-8/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Photo Contest Entry</title>
+    </head>
+    <body>
+        <form>
+            <h1>Photo Contest Entry</h1>
+            <label for="full-name-input">Full Name:</label><br />
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: bad, useCamelCase

Comment (aliramazon) [reply assignments/assignment-8/index.html]: I understand you used br after label. But, you should not be after input. Div can be used to wrap Input and Label. Like, <div>  Label, br, Input </div>

File: assignments/assignment-8/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                required
+            /><br />
+
+            <label for="imageInput">Upload Your Photo:</label
+            ><input
+                type="file"
+                id="imageInput"
+                name="avatar"
+                required
+                accept="image/jpeg, image/jpg, image/webp"
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: I liked it,

File: assignments/assignment-8/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            /><br />
+
+            <label for="checkboxInput"
+                >I agree to the contest terms and conditions</label
+            >
+            <input
+                type="checkbox"
+                id="checkboxInput"
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: There can be multiple checkboxes. So, best one is to use contestCheckboxInput.

File: assignments/assignment-8/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                name="agree"
+                required
+            /><br />
+
+            <label for="countrySelect">Select Your Country</label>
+            <select name="country" id="countrySelect">
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: not consistent with Id selection. here used camelCase

File: assignments/assignment-8/index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                <option value="United States">United States</option>
+                <option value="Canada">Canada</option>
+                <option value="UK">UK</option>
+                <option value="Australia">Australia</option></select
+            ><br />
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: for now, each Select should be wrapped in div

File: assignments/assignment-8/index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+            <label for="description"
+                >Write a brief description or story about your photo</label
+            ><br />
+            <textarea
+                name="description"
+                id="description"
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: descriptionTextarea

File: assignments/assignment-8/index.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                cols="30"
+                rows="10"
+            ></textarea>
```
Comment (aliramazon) [inline assignments/assignment-8/index.html]: required

**Review by aliramazon — CHANGES_REQUESTED** [review]

File: assignments/assignment-8/index.html (hunk 4 above)
Comment (durdonadev) [reply assignments/assignment-8/index.html]: I used "countrySelect" as camalCase for id. I didn't catch what you mention, sorry?

**Comment by aliramazon** [comment]
Approved it.

**Review by aliramazon — APPROVED** [review]

**Review by aliramazon — APPROVED** [review]

**Pull request merged**